### DISCUSSION TITLE
Possibility to manage all active sessions

### DISCUSSION THREAD

**Discussion by perostlin** [supabase/supabase#26826] [Feature Requests]
I would like the possibility to manage sessions currently logged in towards a account (user).

So say I have a account and I sign in with that account on my MacBook and then on my iPhone, I would like to be able to manage these via SDKs.

Kinda like how Netflix does it, I as a user can see all sessions signed in towards my account and I can view, edit or delete them, which will sign the device out from my account.

Then when someone signs in from a new device, in the email (e.g magic link), I will be able to see something like this

<img width="265" alt="image" src="https://github.com/supabase/supabase/assets/20464832/6ef4e117-9639-4865-9057-1a1b1a4ff6cf">

**Comment by guillempuche** [comment]
Also requested in https://github.com/orgs/supabase/discussions/26863
